src/libstd/env.rs

```diff
@@ -68,15 +68,17 @@ pub fn set_current_dir<P: AsRef<Path>>(p: P) -> io::Result<()> {
 
 /// An iterator over a snapshot of the environment variables of this process.
 ///
-/// This iterator is created through `std::env::vars()` and yields `(String,
-/// String)` pairs.
+/// This structure is created through the [`std::env::vars`] function.
```

Comment:
How so? "yields `(String, String)` pairs." isn't saying anything more than what's displayed in the `Iterator` section of `Env`.